examples/byovnet_privateendpoints/main.tf

```diff
@@ -114,6 +113,11 @@ module "azureml" {
       inherit_lock                  = false
     }
   }
-
+  application_insights = {
+    create_new = true
+  }
+  log_analytics_workspace = {
+    create_new = true
+  }
```

Comment:
Shuld we actually create those if what we are trying to show in this example it is private endpoints ? I jsut don't want us to cause the confusion of someone thinking they need this for private endpoints to work.

Reply:
I believe these are created by default when you create AML in azure, so assuming this is necessary (either new or existing)